x-pack/platform/plugins/shared/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -428,6 +432,8 @@ export const AnomalyTimeline: FC<AnomalyTimelineProps> = React.memo(
                         defaultMessage: 'Actions',
                       })}
                       color="text"
+                      display="base"
```

Comment:
Should we apply the same styling to the button for the actions on the anomaly charts?

<img width="377" alt="Screenshot 2024-12-17 at 13 10 49" src="https://github.com/user-attachments/assets/699a2701-7c5f-450f-acfb-d52810be9357" />

Reply:
Good point, missed that one!
Done in: [#e9108d6](https://github.com/elastic/kibana/pull/203813/commits/e9108d64b5f0fcfc8b0c261062126341e5583db5)